tools/make_update_yml.py

```diff
@@ -0,0 +1,179 @@
+            '5. Power cycle the board. It will now enumerate and mount as `DAPLINK` or the name of the board.'
+        ]),
+        ('linux', [
+            '1. Download the firmware file.',
+            '2. While holding down the boards reset button, connect the boards USB debug port to the computer. It should enumerate as `BOOTLOADER` or `MAINTENANCE`. For boards that enumerate as `BOOTLOADER` [see our blog to determine if an update for the DAPLink bootloader is available.](https://os.mbed.com/blog/entry/DAPLink-bootloader-update/)',
+            '3. In a terminal execute',
+            '   - `cp <path to firmware file> <MAINTENANCE> && sync`',
+            '   - Note: make sure to change `MAINTENANCE` to the name of the mount point of the drive on your system.',
+            '4. Power cycle the board. It will now enumerate and mount as `DAPLINK` or the name of the board.'
+        ])
+    ]),
+    'lpc11u35':InstructionList([
+        ('windows', [
+            '1. Download the firmware file.',
+            '2. While holding down the boards reset button, connect the boards USB debug port to the computer. It should enumerate and mount as `CRP DISABLD`',
+            '3. Delete the file named `firmware.bin`',
+            '4. Wait for the file copy operation to complete.',
+            '5. Power cycle the board. It will now enumerate and mount as `DAPLINK` or the name of the board.'
+        ]),
+        ('osx', [
+            '1. Download the firmware file.',
+            '2. While holding down the boards reset button, connect the boards USB debug port to the computer. It should enumerate and mount as `CRP DISABLD`',
+            '3. Delete the file named `firmware.bin`',
+            '4. In a terminal execute',
+            '   - `sudo mount -u -w -o sync /Volumes/CRP\ DISABLD ; rm /Volumes/CRP\ DISABLD/firmware.bin && cp -X <path to firmware file> /Volumes/CRP\ DISABLD/`',
+            '5. Wait for the file copy operation to complete.',
+            '6. Power cycle the board. It will now enumerate and mount as `DAPLINK` or the name of the board.'
+        ]),
+        ('linux', [
+            '1. Download the firmware file.',
+            '2. While holding down the boards reset button, connect the boards USB debug port to the computer. It should enumerate and mount as `CRP DISABLD`',
+            '3. Delete the file named `firmware.bin`',
+            '4. In a terminal execute',
+            '   - `cp <path to firmware file> <CRP DISABLD> && sync`',
+            '   - Note: make sure to change `CRP DISABLD` to the name of the mount point on your system.',
+            '5. Power cycle the board. It will now enumerate and mount as `DAPLINK` or the name of the board.'
+        ])
+    ])
+
+}
+
+
+def string_writer(element):
+	if type(element) is str:
+		return element
+	elif type(element) is int:
+		return "_0x%04i" % element
+	else:
+		return None
+
+
+def log_string (exeption, type, element):
```

Comment:
Instead of defining your own log function, use the `logging` package.

Comments that don't matter given the previous sentence: "exeption" is misspelled ("exception"). I would also use a different term than "exception" since that implies a real exception object, where the parameter here is an error message.